main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-tools/go-steputils/input"
+)
+
+func main() {
+	//Validate inputs
+	if err := input.ValidateIfPathExists(os.Getenv("file_path")); err != nil {
```

Comment:
wire out `os.Getenv("file_path")` to a variable, to do not duplicate os.Getenv:

`filePath := os.Getenv("file_path")`